src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,11 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Ball firstBall = new Lottery().getRandomBall();
+        Ball secondBall = new Lottery().getRandomBall();
+        Ball thirdBall = new Lottery().getRandomBall();
+        System.out.println(firstBall.toString());
+        System.out.println(secondBall.toString());
+        System.out.println(thirdBall.toString());
```

Comment:
![image](https://user-images.githubusercontent.com/81781479/189142921-7ae69ef8-6fc0-4e97-bef4-140ba028c343.png)

Reply:
![image](https://user-images.githubusercontent.com/81781479/189143653-5b0d6b01-cfb4-4284-ba87-449ebc1a544e.png)
you can use just 1 lottery